samples/debug/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _debug_sample:
```

Comment:
samples/debug is too broad, please put this under samples/debug/gdbstub..

Reply:
ack